docs/team.rst

```diff
@@ -0,0 +1,128 @@
+Read the Docs Team
+==================
+
+readthedocs.org is the largest open source documentation hosting service.
+It's provided as a free service to the open source community,
+and is worked on by a community of volunteers that we're hoping to expand!
+We currently serve over 20,000,000 pageviews a month,
+and we hope to maintain a reliable and stable hosting platform for years to come.
+
+There are three major parts of this work:
+
+* Support
+* Operations
+* Development
+
+I'll talk about each below,
+as well as mentioning who is on the team.
```

Comment:
This block is superfluous, the structure of each section is obvious by looking at it. Linking the list above to the headings would reduce the need for this.